Tests/UIKitTests/UIImageExtensionsTests.swift

```diff
@@ -47,6 +47,16 @@ final class UIImageExtensionsTests: XCTestCase {
         XCTAssertNil(UIImage().compressed())
     }
 
+        func testCompressedData() {
+        let bundle = Bundle.init(for: UIImageExtensionsTests.self)
```

Comment:
```suggestion
        let bundle = Bundle(for: UIImageExtensionsTests.self)
```

Reply:
Btw, if that's the preferred style, should I do another PR and just refactor the other Xyz.init(...)? .-.

Reply:
I wouldn't worry about it...